wrappers/python/nntile/nntile_core.cc

```diff
@@ -733,10 +733,12 @@ void def_mod_tensor(py::module_ &m)
     m.def("gather_bool", &gather<bool_t>);
     m.def("gather_bf16", &gather<bf16_t>);
 
+    m.def("copy_intersection_async_bool", &copy_intersection_async<bool_t>);
     m.def("copy_intersection_async_fp64", &copy_intersection_async<fp64_t>);
     m.def("copy_intersection_async_fp32", &copy_intersection_async<fp32_t>);
     m.def("copy_intersection_async_int64", &copy_intersection_async<nntile::int64_t>);
 
+    m.def("copy_intersection_bool", &copy_intersection<bool_t>);
```

Comment:
Tensor function `copy_intersection<bool_t>` is not defined, actually. How this is even compiled?

Reply:
done. Seems to be faulty recompile by me cause the issue